content/2017/06/27/susy3.rst

```diff
@@ -0,0 +1,872 @@
+public: yes
+author: miriam
+location: 'susysite-articles'
+template: 'susysite/layout.html'
+tags: [Susy, Susy3, CSS, Sass, Grids, Code]
+headline:
+  - type: 'Tutorial'
+    tagline: 'Make grid systems your fallback plan'
+image:
+  - src: 'blog/2017/spread/susy-spread.jpg'
+summary: |
+  **We're excited to introduce Susy 3.0**,
+  a major update to our popular grid-math calculator –
+  now more focused and flexible than ever.
+  Susy was designed to make layout math easy,
+  without forcing you into generic patterns
+  and ugly markup.
+  But grid systems are on the way out,
+  replaced by real CSS layout specs
+  that live in the browser.
+  With Susy3,
```

Comment:
Maybe make this 'Susy Three' to be consistent with the Susy section of OddSite? Or switch Susy section of OddSite to Susy3?